skdownscale/pointwise_models/bcsd.py

```diff
@@ -5,31 +5,56 @@
 from sklearn.utils.validation import check_is_fitted
 
 from .base import TimeSynchronousDownscaler
-from .utils import QuantileMapper
-
-
-def MONTH_GROUPER(x):
-    return x.month
+from .groupers import DAY_GROUPER, MONTH_GROUPER, PaddedDOYGrouper
+from .utils import QuantileMapper, check_datetime_index, ensure_samples_features
 
 
 class BcsdBase(TimeSynchronousDownscaler):
     """ Base class for BCSD model.
     """
 
-    _fit_attributes = ['y_climo_', 'quantile_mappers_']
+    _fit_attributes = ["y_climo_", "quantile_mappers_"]
     _timestep = 'M'
 
-    def __init__(self, time_grouper=MONTH_GROUPER, return_anoms=True, qm_kwargs={}):
+    def __init__(
+        self,
+        time_grouper=MONTH_GROUPER,
+        climate_trend_grouper=DAY_GROUPER,
+        return_anoms=True,
+        **qm_kwargs):
+        
         self.time_grouper = time_grouper
+        self.climate_trend_grouper = climate_trend_grouper
+        self.climate_trend = MONTH_GROUPER
         self.return_anoms = return_anoms
         self.qm_kwargs = qm_kwargs
```

Comment:
```suggestion
    def __init__(
        self,
        time_grouper=MONTH_GROUPER,
        climate_trend_grouper=DAY_GROUPER,
        return_anoms=True,
        qm_kwargs={}):
        
        self.time_grouper = time_grouper
        self.climate_trend_grouper = climate_trend_grouper
        self.climate_trend = MONTH_GROUPER
        self.return_anoms = return_anoms
        self.qm_kwargs = qm_kwargs
```

Changing this back should fix a few of your failing tests.

Reply:
good catch @jhamman, thanks